lib/SILOptimizer/SILCombiner/SILCombiner.h

```diff
@@ -295,9 +295,12 @@ class SILCombiner :
 private:
   FullApplySite rewriteApplyCallee(FullApplySite apply, SILValue callee);
 
-  SILInstruction *createApplyWithConcreteType(FullApplySite Apply,
-                                              const ConcreteExistentialInfo &CEI,
-                                              SILBuilderContext &BuilderCtx);
+  bool canReplaceArg(FullApplySite Apply, const ConcreteExistentialInfo &CEI,
+                     unsigned ArgIdx);
+  SILInstruction *createApplyWithConcreteType(
+      FullApplySite Apply,
+      llvm::SmallDenseMap<unsigned, const ConcreteExistentialInfo *, 8> &CEIs,
```

Comment:
Passing SmallDenseMapImpl instead would allow you to avoid hard-coding the size here. Also you should make it 'const'.

Reply:
Which header file has this SmallDenseMapImpl?